icons/rollup/rollup-line.svg

```diff
@@ -0,0 +1 @@
0.4 80.4c-.3-.7-.1-1.5.5-1.8 12.3-7.1 20.6-20.3 20.6-35.6 0-7.3-1.9-14.2-5.3-20.1-.3-.5-.6-1-.8-1.4v-.1C98.3 9.9 85.6 2.2 71.2 2H22zm0 1h49.2c11.9.2 22.5 5.6 29.7 14l-.1-.1-.1-.1c-.3-.2-.5-.4-.8-.6-.1-.1-.2-.2-.4-.3-.1-.1-.2-.1-.2-.2-.1-.1-.3-.2-.4-.3-.1 0-.1-.1-.2-.1-.1-.1-.3-.2-.4-.3-.3-.2-.7-.4-1-.6-.1 0-.1-.1-.2-.1-.4-.2-.7-.4-1.1-.6-.1 0-.2-.1-.3-.1-.1-.1-.2-.1-.4-.2s-.3-.2-.5-.2c-.1 0-.2-.1-.3-.1-.1 0-.1-.1-.2-.1-.4-.1-.7-.3-1.1-.4h-.1c-.1 0-.2-.1-.3-.1h-.1c-.3-.1-.7-.2-1-.3-.1 0-.2-.1-.4-.1-.1 0-.2-.1-.4-.1-.2-.1-.4-.1-.6-.2-.1 0-.1 0-.2-.1-.1 0-.2 0-.2-.1-.4-.1-.8-.2-1.2-.2h-.4c-.5-.1-.9-.1-1.4-.2h-.4c-.2 0-.4 0-.6-.1h-.4c-.1 0-.1 0 0 0H84c-.9 0-1.7 0-2.5.1h-.3c-1.3.1-2.5.2-3.6.5-.8.2-1.6.4-2.3.7-1.5.6-3 1.4-4.5 2.5-.7.6-1.5 1.2-2.2 1.9l-.3.3c-.3.3-.7.7-1 1.1-.7.8-1.4 1.6-2.2 2.5-.1.1-.2.3-.3.4-.6.8-1.2 1.6-1.9 2.5-.3.4-.6.9-1 1.4-.5.7-1 1.5-1.5 2.2-.3.5-.6 1-1 1.5-.7 1.1-1.4 2.2-2.1 3.4-3.5 5.7-7.8 13.2-12.9 22.4-2.6 4.7-5.4 9.7-8.5 15.5 0 .1-.1.2-.1.3-1.5 2.7-2.8 5.2-4.1 7.6-5.1 9.7-8.5 17.2-10.8 23V4.4c-.3-.8.3-1.4 1.1-1.4zm61.9 9.2c.8 0 1.5 0 2.3.1h.3c5.6.6 11 3 15.3 6.9-3.1-1.9-6.7-3.3-10.3-3.9l-1.8-.3h-.2c-.5-.1-1.1-.1-1.6-.1h-2.4c-.9.1-1.7.2-2.6.3-.7.1-1.3.3-2 .5-3.8 1.1-6.9 3.4-8.4 7.1-2.5 5.9-.4 12.2 3.1 16.8 1.7 2.3 3.8 4.2 5.8 5.5.2.1.3.2.5.3h.1c.1.1.2.1.3.1.3.2.6.3 1 .5.3.1.7.3 1 .4.1 0 .2.1.3.1l1.2.3h.8c.2 0 .4 0 .6-.1.3 0 .5-.1.7-.2l.6-.3c.4-.2.7-.5.9-.8.5-.6.8-1.4 1-2.3.3-1.7 0-3.6-.5-5.6.3.7.9 1.7 1.1 2.2.7 1.8.9 3.2.7 4.3v.2c-.1.4-.2.8-.4 1.2-.8 1.6-2.6 3.2-5.6 5.7-2.9 2.4-6.9 5.7-12 10.6-10.2 9.8-20.4 24.9-28.5 37.8-4 6.4-7.6 12.3-10.2 16.7-1.3 2.2-2.5 4-3.4 5.3-.4.7-.8 1.2-1.1 1.6-.3.4-.6.6-.6.6-2.4 1.4-6.3 1.7-8.9.1l-.1-.1c-.2-.1-.3-.2-.5-.4l-.1-.1-.6-.6c-.3-.3-.6-.7-.9-1.1-.5-.7-.8-1.5-1-2.3-.1-.3-.1-.7-.2-1.1-.2-1.4-.1-3.4.5-6.1.6-2.7 1.7-6.2 3.4-10.7.5-1.3 1-2.6 1.7-4.1 2-4.6 4.6-10.1 7.9-16.5 1.6-3.2 3.4-6.5 5.4-10.2 0-.1.1-.1.1-.2 3.2-5.8 5.9-10.9 8.5-15.6 1.9-3.4 3.7-6.7 5.3-9.6 3.8-6.7 7-12 9.7-16.3l.6-.9c.7-1 1.3-2 1.9-2.9l.6-.9c.8-1.1 1.5-2.1 2.3-3 .7-.9 1.4-1.6 2-2.3.5-.5.9-1 1.4-1.4.7-.7 1.4-1.3 2.2-1.8 1.4-1.1 2.8-1.8 4.3-2.3.6-.2 1.1-.4 1.7-.5.3-.1.7-.2 1.1-.2.3-.1.6-.1.9-.2.6-.1 1.2-.1 1.8-.2.9.1 1.9 0 3 0zm3.9 3.7c6.4.3 13.3 3 17.6 7.4 3.3 5.8 5.1 12.5 5.1 19.6 0 14.9-8.1 27.8-20.1 34.7-1.1.6-1.5 2-.9 3.1l20.8 41.1c.5.9-.2 2-1.2 2H31l.2-.2c.3-.4.7-1 1.2-1.6.9-1.3 2-3.1 3.3-5.2 0-.1.1-.1.1-.2 2.7-4.4 6.2-10.3 10.2-16.7 8-12.6 18.3-27.7 28.3-37.3 5.1-4.8 9-8.1 11.9-10.5 2.9-2.5 4.8-4.1 5.8-6 .2-.5.4-.9.5-1.4.3-1.5.1-3.1-.7-5.1-1-2.7-3-6.3-5.8-11.5-.1-.2-.2-.3-.4-.3s-.3.1-.4.2c0 0-3.4 4.8-8.3 11.9-.2-.2-.4-.4-.6-.7-1.8-2.4-3.2-5.2-3.7-8.1 0-.3-.1-.5-.1-.8-.3-2.3-.1-4.6.9-6.9 2-4.7 6.7-7 12.1-7.3.7-.2 1.5-.2 2.3-.2zm-2.3 13.2c.5.9 2 4 3 7.7.6 2.1 1 4.3.8 6-.1.8-.3 1.5-.7 2-.4.5-.9.8-1.7 1-.3.1-.6.1-1 0-.3 0-.7-.1-1.1-.2-.1 0-.1 0-.2-.1-.3-.1-.7-.2-1.1-.4-.1 0-.2-.1-.3-.1-.3-.2-.7-.3-1-.5-.2-.1-.5-.3-.7-.5-.1-.1-.3-.2-.4-.3-1.2-.9-2.4-1.9-3.5-3.2 4.6-6.7 7.6-10.9 7.9-11.4zM18 112.2z" style="fill:#f33"/></svg>
```

Comment:
Remove `style` and `xml:space`. 🙂
This suggestion does not change the appearance of the icon, here's the preview:
<div align="center">
  <img width="128" src="https://user-images.githubusercontent.com/76992016/201240010-1574a380-f453-4b30-9cdd-523a0db762f6.svg">
</div>

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M22 2c-1.3 0-2.4 1.1-2.4 2.4v99.2c-1.3 3.4-2.1 6.2-2.6 8.5v.1c-.6 2.7-.7 4.8-.5 6.3 0 .3.1.6.1.9.1.8.4 1.5.8 2.1.2.4.4.7.6 1 .3.4.6.7.8 1l.7.7c.1 0 .1.1.1.1.2.1.4.3.6.4l.1.1h.1c2.8 1.7 6.6 1.5 9.3.2h79.4c1.8 0 2.9-1.9 2.1-3.5L90.4 80.4c-.3-.7-.1-1.5.5-1.8 12.3-7.1 20.6-20.3 20.6-35.6 0-7.3-1.9-14.2-5.3-20.1-.3-.5-.6-1-.8-1.4v-.1C98.3 9.9 85.6 2.2 71.2 2H22zm0 1h49.2c11.9.2 22.5 5.6 29.7 14l-.1-.1-.1-.1c-.3-.2-.5-.4-.8-.6-.1-.1-.2-.2-.4-.3-.1-.1-.2-.1-.2-.2-.1-.1-.3-.2-.4-.3-.1 0-.1-.1-.2-.1-.1-.1-.3-.2-.4-.3-.3-.2-.7-.4-1-.6-.1 0-.1-.1-.2-.1-.4-.2-.7-.4-1.1-.6-.1 0-.2-.1-.3-.1-.1-.1-.2-.1-.4-.2s-.3-.2-.5-.2c-.1 0-.2-.1-.3-.1-.1 0-.1-.1-.2-.1-.4-.1-.7-.3-1.1-.4h-.1c-.1 0-.2-.1-.3-.1h-.1c-.3-.1-.7-.2-1-.3-.1 0-.2-.1-.4-.1-.1 0-.2-.1-.4-.1-.2-.1-.4-.1-.6-.2-.1 0-.1 0-.2-.1-.1 0-.2 0-.2-.1-.4-.1-.8-.2-1.2-.2h-.4c-.5-.1-.9-.1-1.4-.2h-.4c-.2 0-.4 0-.6-.1h-.4c-.1 0-.1 0 0 0H84c-.9 0-1.7 0-2.5.1h-.3c-1.3.1-2.5.2-3.6.5-.8.2-1.6.4-2.3.7-1.5.6-3 1.4-4.5 2.5-.7.6-1.5 1.2-2.2 1.9l-.3.3c-.3.3-.7.7-1 1.1-.7.8-1.4 1.6-2.2 2.5-.1.1-.2.3-.3.4-.6.8-1.2 1.6-1.9 2.5-.3.4-.6.9-1 1.4-.5.7-1 1.5-1.5 2.2-.3.5-.6 1-1 1.5-.7 1.1-1.4 2.2-2.1 3.4-3.5 5.7-7.8 13.2-12.9 22.4-2.6 4.7-5.4 9.7-8.5 15.5 0 .1-.1.2-.1.3-1.5 2.7-2.8 5.2-4.1 7.6-5.1 9.7-8.5 17.2-10.8 23V4.4c-.3-.8.3-1.4 1.1-1.4zm61.9 9.2c.8 0 1.5 0 2.3.1h.3c5.6.6 11 3 15.3 6.9-3.1-1.9-6.7-3.3-10.3-3.9l-1.8-.3h-.2c-.5-.1-1.1-.1-1.6-.1h-2.4c-.9.1-1.7.2-2.6.3-.7.1-1.3.3-2 .5-3.8 1.1-6.9 3.4-8.4 7.1-2.5 5.9-.4 12.2 3.1 16.8 1.7 2.3 3.8 4.2 5.8 5.5.2.1.3.2.5.3h.1c.1.1.2.1.3.1.3.2.6.3 1 .5.3.1.7.3 1 .4.1 0 .2.1.3.1l1.2.3h.8c.2 0 .4 0 .6-.1.3 0 .5-.1.7-.2l.6-.3c.4-.2.7-.5.9-.8.5-.6.8-1.4 1-2.3.3-1.7 0-3.6-.5-5.6.3.7.9 1.7 1.1 2.2.7 1.8.9 3.2.7 4.3v.2c-.1.4-.2.8-.4 1.2-.8 1.6-2.6 3.2-5.6 5.7-2.9 2.4-6.9 5.7-12 10.6-10.2 9.8-20.4 24.9-28.5 37.8-4 6.4-7.6 12.3-10.2 16.7-1.3 2.2-2.5 4-3.4 5.3-.4.7-.8 1.2-1.1 1.6-.3.4-.6.6-.6.6-2.4 1.4-6.3 1.7-8.9.1l-.1-.1c-.2-.1-.3-.2-.5-.4l-.1-.1-.6-.6c-.3-.3-.6-.7-.9-1.1-.5-.7-.8-1.5-1-2.3-.1-.3-.1-.7-.2-1.1-.2-1.4-.1-3.4.5-6.1.6-2.7 1.7-6.2 3.4-10.7.5-1.3 1-2.6 1.7-4.1 2-4.6 4.6-10.1 7.9-16.5 1.6-3.2 3.4-6.5 5.4-10.2 0-.1.1-.1.1-.2 3.2-5.8 5.9-10.9 8.5-15.6 1.9-3.4 3.7-6.7 5.3-9.6 3.8-6.7 7-12 9.7-16.3l.6-.9c.7-1 1.3-2 1.9-2.9l.6-.9c.8-1.1 1.5-2.1 2.3-3 .7-.9 1.4-1.6 2-2.3.5-.5.9-1 1.4-1.4.7-.7 1.4-1.3 2.2-1.8 1.4-1.1 2.8-1.8 4.3-2.3.6-.2 1.1-.4 1.7-.5.3-.1.7-.2 1.1-.2.3-.1.6-.1.9-.2.6-.1 1.2-.1 1.8-.2.9.1 1.9 0 3 0zm3.9 3.7c6.4.3 13.3 3 17.6 7.4 3.3 5.8 5.1 12.5 5.1 19.6 0 14.9-8.1 27.8-20.1 34.7-1.1.6-1.5 2-.9 3.1l20.8 41.1c.5.9-.2 2-1.2 2H31l.2-.2c.3-.4.7-1 1.2-1.6.9-1.3 2-3.1 3.3-5.2 0-.1.1-.1.1-.2 2.7-4.4 6.2-10.3 10.2-16.7 8-12.6 18.3-27.7 28.3-37.3 5.1-4.8 9-8.1 11.9-10.5 2.9-2.5 4.8-4.1 5.8-6 .2-.5.4-.9.5-1.4.3-1.5.1-3.1-.7-5.1-1-2.7-3-6.3-5.8-11.5-.1-.2-.2-.3-.4-.3s-.3.1-.4.2c0 0-3.4 4.8-8.3 11.9-.2-.2-.4-.4-.6-.7-1.8-2.4-3.2-5.2-3.7-8.1 0-.3-.1-.5-.1-.8-.3-2.3-.1-4.6.9-6.9 2-4.7 6.7-7 12.1-7.3.7-.2 1.5-.2 2.3-.2zm-2.3 13.2c.5.9 2 4 3 7.7.6 2.1 1 4.3.8 6-.1.8-.3 1.5-.7 2-.4.5-.9.8-1.7 1-.3.1-.6.1-1 0-.3 0-.7-.1-1.1-.2-.1 0-.1 0-.2-.1-.3-.1-.7-.2-1.1-.4-.1 0-.2-.1-.3-.1-.3-.2-.7-.3-1-.5-.2-.1-.5-.3-.7-.5-.1-.1-.3-.2-.4-.3-1.2-.9-2.4-1.9-3.5-3.2 4.6-6.7 7.6-10.9 7.9-11.4zM18 112.2z" fill="#f33"/></svg>

```